docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -172,32 +172,166 @@ in create table statement. ``ReplicatedMergeTree`` engine is not yet supported.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps ClickHouse types to the corresponding Trino types according
+to the following table:
+
+.. list-table:: ClickHouse type to Trino type mapping
+  :widths: 30, 25, 50
+  :header-rows: 1
+
+  * - ClickHouse type
+    - Trino type
+    - Notes
+  * - ``Int8``
+    - ``TINYINT``
+    - ``TINYINT``, ``BOOL``, ``BOOLEAN`` and ``INT1`` are aliases of ``Int8``
+  * - ``Int16``
+    - ``SMALLINT``
+    -  ``SMALLINT`` and ``INT2`` are aliases of ``Int16``
+  * - ``Int32``
+    - ``INTEGER``
+    - ``INT``, ``INT4`` and ``INTEGER`` are aliases of ``Int32``
+  * - ``Int64``
+    - ``BIGINT``
+    - ``BIGINT`` is an alias of ``Int64``
+  * - ``UInt8``
+    - ``SMALLINT``
+    -
+  * - ``UInt16``
+    - ``INTEGER``
+    -
+  * - ``UInt32``
+    - ``BIGINT``
+    -
+  * - ``UInt64``
+    - ``DECIMAL(20,0)``
+    -
+  * - ``Float32``
+    - ``REAL``
+    - ``FLOAT`` is an alias of ``Float32``
+  * - ``Float64``
+    - ``DOUBLE``
+    - ``DOUBLE`` is an alias of ``Float64``
+  * - ``Decimal``
+    - ``DECIMAL``
+    -
+  * - ``FixedString``
+    - ``VARBINARY``
+    - Enabling ``clickhouse.map-string-as-varchar`` config property changes the
+      mapping to ``VARCHAR``
+  * - ``String``
+    - ``VARBINARY``
+    - Enabling ``clickhouse.map-string-as-varchar`` config property changes the
+      mapping to ``VARCHAR``
+  * - ``Date``
+    - ``DATE``
+    -
+  * - ``DateTime``
+    - ``TIMESTAMP``
+    -
+  * - ``IPv4``
+    - ``IPADDRESS``
+    -
+  * - ``IPv6``
+    - ``IPADDRESS``
+    -
+  * - ``Enum8``
+    - ``VARCHAR``
+    -
+  * - ``Enum16``
+    - ``VARCHAR``
+    -
+  * - ``UUID``
+    - ``UUID``
+    -
+
+No other types are supported.
+
+Trino type to ClickHouse type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps Trino types to the corresponding ClickHouse types according
+to the following table:
+
+.. list-table:: Trino type to ClickHouse type mapping
+  :widths: 30, 20, 50
+  :header-rows: 1
+
+  * - Trino type
+    - ClickHouse type
+    - Notes
+  * - ``TINYINT``
+    - ``Int8``
+    - ``TINYINT``, ``BOOL``, ``BOOLEAN`` and ``INT1`` are aliases of ``Int8``
+  * - ``SMALLINT``
+    - ``Int16``
+    -  ``SMALLINT`` and ``INT2`` are aliases of ``Int16``
+  * - ``INTEGER``
+    - ``Int32``
+    - ``INT``, ``INT4`` and ``INTEGER`` are aliases of ``Int32``
+  * - ``BIGINT``
+    - ``Int64``
+    - ``BIGINT`` is an alias of ``Int64``
+  * - ``SMALLINT``
+    - ``UInt8``
+    -
+  * - ``INTEGER``
+    - ``UInt16``
+    -
+  * - ``BIGINT``
+    - ``UInt32``
+    -
+  * - ``DECIMAL(20,0)``
+    - ``UInt64``
+    -
+  * - ``REAL``
+    - ``Float32``
+    - ``FLOAT`` is an alias of ``Float32``
+  * - ``DOUBLE``
+    - ``Float64``
+    - ``DOUBLE`` is an alias of ``Float64``
+  * - ``DECIMAL``
+    - ``Decimal``
+    -
+  * - ``VARBINARY``
+    - ``FixedString``
+    - Enabling ``clickhouse.map-string-as-varchar`` config property changes the
+      mapping to ``VARCHAR``
```

Comment:
Remove as duplicate of subsequent entries.

Reply:
?

Reply:
Done in commit [d563ed5](https://github.com/trinodb/trino/pull/13578/commits/d563ed5b5935c6b67bc57e98b11d86cc474582db)